include/rwahs/tasks/deploy

```diff
@@ -73,7 +73,7 @@ function deploy_runTask {
         error="Required parameter 'component' not specified for 'deploy' task."
     elif [ ! -f "${toolsDir}/config/${programName}/components/${component}" ]; then
         error="Specified component '${component}' does not exist, see below for valid values."
-    elif [ ! -z "${environment}" ]; then
+    elif [ -z "${environment}" ]; then
```

Comment:
I remember seeing this and thinking is that right for `-z` and then forgot to follow up.